accounts/kv_sql_migration_test.go

```diff
@@ -0,0 +1,339 @@
+package accounts
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/lightninglabs/lightning-terminal/db"
+	"github.com/lightningnetwork/lnd/clock"
+	"github.com/lightningnetwork/lnd/fn"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/sqldb"
+	"github.com/stretchr/testify/require"
+)
+
+// TestAccountStoreMigration tests the migration of account store from a bolt
+// backed to a SQL database. Note that this test does not attempt to be a
+// complete migration test.
+func TestAccountStoreMigration(t *testing.T) {
+	t.Parallel()
+
+	ctxc, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
+
+	// First create a shared Postgres instance so we don't spawn a new
+	// docker container for each test.
+	pgFixture := db.NewTestPgFixture(
+		t, db.DefaultPostgresFixtureLifetime, true,
+	)
+	t.Cleanup(func() {
+		pgFixture.TearDown(t)
+	})
+
+	makeSQLDB := func(t *testing.T, clock *clock.TestClock,
+		sqlite bool) (*SQLStore, *db.TransactionExecutor[SQLQueries]) {
+
+		var store *SQLStore
+
+		if sqlite {
+			store = NewSQLStore(db.NewTestSqliteDB(t).BaseDB, clock)
+		} else {
+			store = NewSQLStore(
+				db.NewTestPostgresDB(t).BaseDB, clock,
+			)
```

Comment:
see other comment about using the existing infrastructure in LiT that uses build flags instead

Reply:
Thanks, updated!